portfolio/src/main/java/com/google/sps/servlets/LogInServlet.java

```diff
@@ -28,7 +28,18 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json;");
     UserService userService = UserServiceFactory.getUserService();
-    boolean loggedIn = userService.isUserLoggedIn();
-    response.getWriter().println(new Gson().toJson(loggedIn));
+    List<String> loginInfo = new ArrayList<String>();
+    String loggedIn = Boolean.toString(userService.isUserLoggedIn());
+    loginInfo.add(loggedIn);
+    if (loggedIn.equals("false")) {
+      String urlToRedirectToAfterUserLogsIn = "/";
+      String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+      loginInfo.add(loginUrl);
+    } else {
+      String urlToRedirectToAfterUserLogsOut = "/";
```

Comment:
Jack had me put this as a class final variable called ROOT_URL, and then using ROOT_URL instead of urlToRedirectToAfterUserLogsIn/Out